packages/core/backend/sirius-components-collaborative/src/main/resources/schema/core.graphqls

```diff
@@ -26,6 +26,7 @@ type EditingContext {
   queryBasedObject(query: String!, variableName: String): Object
   queryBasedObjects(query: String!, variableName: String): [Object]
   actions: EditingContextActionsConnection!
+  textBasedObjects(query: String!): [Object]!
```

Comment:
I didn't paginate the result, since it was not done in the query API I assumed it was fine. Let me know if I should do it

Reply:
Btw this is not related to this PR, but the fields `queryBasedXXX` don't have a data fetcher. I don't know if this is on purpose, but I prefer to mention it just in case

Reply:
- query.graphls is for the query view, I'll just move this to the omnibox.graphqls file
- The pagination can be added later
- Yes I know that there is an issue with the queryBasedXxx